audio/qjackctl/files/0002-Restore-Qt4-support.patch

```diff
@@ -0,0 +1,585 @@
+From 513b39b7188ddb582467a182f56d1a3ab9c38c20 Mon Sep 17 00:00:00 2001
```

Comment:
is this all upstream stuff? Please add a link to the commit(s)/PR where upstream removed support for Qt4 as presumably you just reverted that?

Reply:
It was cherry picked, but a large part was from https://github.com/rncbc/qjackctl/commit/5a2b27193553b794f9b54b867ae8c88076016692

The current code is simply incoherent: for example, it check for Qt < 4.5 (!), but uses `<QRegularExpression>` unconditionally (it was introduced in Qt5): https://github.com/rncbc/qjackctl/blob/ed50caeb4d71d38b3eccd59617da5c1219b9929c/src/qjackctlMainForm.cpp

And no, I am not gonna spend another two hrs on looking through commit history and classify what was reverted and what was just fixed directly.

I do not see anything powerpc-specific here or pre-release 10.6-specific, so the fix is of general relevance. If you have fixed Qt5 5.15 for 10.6 x86, please let me know.
However, I am not gonna try convincing anyone anymore. Please feel free to close the PR if you do not want it merged. Fine for me.